tests/tests.bats

```diff
@@ -644,6 +645,20 @@ xen_expect_abort() {
   expect_success
 }
 
+@test "net_mtu hvt" {
```

Comment:
the test net_mtu, is not present... did you forgot to git add it?

Reply:
Yes, I forgot to add them. They are in https://github.com/Solo5/solo5/pull/606/commits/4064e8f432f1c1fda30cbe2f1db1e03390ca3d31